releasing/cloudbuild-local.sh

```diff
@@ -27,7 +27,7 @@ cp releasing/cloudbuild.yaml $config
 # Add the --snapshot flag to suppress the
 # github release and leave the build output
 # in the kustomize/dist directory.
-sed -i "s|# - '--snapshot|- '--snapshot|" $config
+sed -i "" "s|# - '--snapshot|- '--snapshot|" $config
```

Comment:
This is a linux vs mac sed difference. Both active maintainers currently use macs.